client/app/scripts/superdesk-desks/views/aggregate-widget.html

```diff
@@ -2,12 +2,25 @@
     <div class="search-container">
         <div class="search-box">
             <input type="text" placeholder="{{ 'Search'|translate }}" ng-model="query">
+
+            <div class="dropdown more-actions" dropdown >
+                <button class="dropdown-toggle" dropdown-toggle><i class="icon-dots-vertical"></i></button>
+                <div class="dropdown-menu pull-right">
+                    <ul>
+                        <li>
+                            <button ng-click="modalActive = true">
+                                <i class="icon-settings"></i> {{ :: 'Content settings' | translate }}
+                            </button>
+                        </li>
+                    </ul>
+                </div>
+            </div>
         </div>
 
         <div class="filter-list">
-            
+
         </div>
-        
+
         <div class="content-list">
             <div class="desk"  ng-repeat="desk in desks._items">
                 <div class="stage" ng-repeat="stage in deskStages[desk._id]" ng-show="total" ng-init="open = true">

```

Comment:
Does the list of desk stages get updated (recreated)? If so, it is advisable to add `track by` clause to ng-repeat to potentially save some unnecessary DOM operations (i.e. destroying and re-creating DOM elements corresponding to the same item in the list).